content/en/docs/kitex/Tutorials/basic-feature/fallback.md

```diff
@@ -6,4 +6,216 @@ keywords: ["Kitex", "Fallback"]
+
+## 2.Usage approach
+
+Due to the involvement of business logic, Fallback is only supported through code configuration.
+
+### 2.1 Client-level configuration
+
+```Go
+var opts []client.Option
+opts = append(opts, client.WithFallback(yourFallbackPolicy))
+
+xxxCli := xxxservice.NewClient("target_service", opts...)
+```
+
+### Call-level configuration
+
+```Go
+import (
+"github.com/cloudwego/kitex/pkg/retry"
+bretry "code.byted.org/kite/kitex/pkg/retry"
+)
+
+xxxCli.XXXMethod(ctx, req, callopt.WithFallback(yourFallbackPolicy))
+```
+
+### 2.3 How to configure Fallback Policy
+
+#### 2.3.1 Define your Fallback Func
+
+Kitex provides two ways to define Fallback Func:
+
+1. Use XXXArgs/XXXResult as req/resp parameters, similar to Middleware.
+
+2. Use actual RPC Req/Resp as parameters, similar to Handler's parameter types.
+
+The latter is more intuitive and user-friendly, but it is not compatible with APIs that have multiple request parameters. Therefore, the framework defaults to using the former method.
+
+
+**Use XXXArgs/XXXResult as req/resp parameters**  
+
+Note: You must replace the original return value using result.SetSuccess(yourFallbackResult).
+
+```go
+// Func is the definition for fallback func, which can do fallback both for error and resp.
+// Notice !! The args and result are not the real rpc req and resp, are respectively XXArgs and XXXResult of generated code.
+// setup eg: client.WithFallback(fallback.NewFallbackPolicy(yourFunc))
+type Func func(ctx context.Context, args utils.KitexArgs, result utils.KitexResult, err error) (fbErr error)
+
+// use demo
+client.WithFallback(
+fallback.NewFallbackPolicy(
+func(ctx context.Context, args utils.KitexArgs,
+result utils.KitexResult, err error) (fbErr error) {
+// your fallback logic...
+result.SetSuccess(yourFallbackResult)
+return
+}))
+```
+
+**Use actual RPC Req/Resp as parameters** 
+
+By using the **fallback.UnwrapHelper** provided by Kitex, you can define a Fallback Func with the signature of RealReqRespFunc, whose parameter types are consistent with Handler's req and resp.
+
+Note: If you need to return resp, you need to construct the actual RPC resp as the return value here, and the Helper will call the SetSuccess method to replace the original return value.
+
+```Go
+// RealReqRespFunc is the definition for fallback func with real rpc req as param, and must return the real rpc resp.
+// setup eg: client.WithFallback(fallback.NewFallbackPolicy(fallback.UnwrapHelper(yourRealReqRespFunc)))
+type RealReqRespFunc func(ctx context.Context, req, resp interface{}, err error) (fbResp interface{}, fbErr error)
+
+// use demo
+client.WithFallback(fallback.NewFallbackPolicy(fallback.UnwrapHelper(func(ctx context.Context, req, resp interface{}, err error) (fbResp interface{}, fbErr error) {
+// your fallback logic...
+return fbResp, fbErr
+}))
+```
```

Comment:
pls format all the code mentioned in this doc

Reply:
The code has been formatted, please take a look again.